lib/email/store.rb

```diff
@@ -0,0 +1,40 @@
+# move this require in gemspec
+require 'dalli'
+
+module Email
+  class Store
+    MEMCACHED_NAMESPACE = 'email_store'.freeze
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.